core/src/models/Export.ts

```diff
@@ -291,6 +293,38 @@ export class Export extends CommonModel<Export> {
     }
   }
 
+  @AfterUpdate
+  static async logExport(instance: Export) {
+    if (!process.env.GROUPAROO_EXPORT_LOG) return;
+
+    if (
+      instance.changed("state") &&
+      ["canceled", "failed", "complete"].includes(instance.state)
+    ) {
+      const exportData = {
```

Comment:
Could this be `apiData()` ?

Reply:
I guess there's nothing wrong with having more info! 

Switched over to using apiData in 693b1db.